docs/UserGuide.adoc

```diff
@@ -775,25 +776,28 @@ image::ui-screenshots/uimockup1_online.png[@Online Tab View, 600, 400]
 .Mark's Offline Tab View
 image::ui-screenshots/uimockup1_offline.png[@Offline Tab View, 600, 400]
 
+_{ more coming soon }_
 
-*What is the [[Dashboard]] Dashboard?*
+== Glossary
+This glossary aims to provide a definition for the special vocabulary used in this user guide.
 
-****
+[[annotation]] Annotation::
+Notes or highlights that can be made on <<offline-copy, offline copies>> of a bookmark.
```

Comment:
```suggestion
Notes or highlights of <<offline-copy, offline copies>>.
```
I think don't need to add "can be" because it kind of increase the uncertainty of the definition. (Compare to other definition)

Reply:
Ok!
```suggestion
A note or highlight that is made on an <<offline-copy, offline copy>> of a bookmark.
```